application/src/test/java/net/corda/v5/application/crypto/MerkleTreeFactoryJavaApiTest.java

```diff
@@ -34,9 +34,6 @@ void createTree() {
     void createHashDigest() {
         final DigestAlgorithmName digestAlgorithmName = DigestAlgorithmName.SHA2_256;
         final MerkleTreeHashDigest hashDigest = mock(MerkleTreeHashDigest.class);
-        // TODO: figure out what to do about the default third argument to createHashDigest, not supported in Java
-        // by default. According to our coding standards doc, we should either replace this with a manual
-        // overload. See JIRA CORE-8374.
```

Comment:
@lankydan I removed this as I noticed the JIRA was "Done" and had your name to it. Let me know if that is ok.